File: src/main/java/ru/ioleksiv/telegram/bot/api/model/annotation/validator/impl/ShippingQueryFilterValidator.java
package ru.ioleksiv.telegram.bot.api.model.annotation.validator.impl;

import ru.ioleksiv.telegram.bot.api.model.annotation.validator.CustomValidator;
import ru.ioleksiv.telegram.bot.api.model.annotation.validator.FilterValidator;
import ru.ioleksiv.telegram.bot.api.model.objects.payments.ShippingQuery;

public class ShippingQueryFilterValidator extends FilterValidator<ShippingQuery> {

    public ShippingQueryFilterValidator(CustomValidator<ShippingQuery> validator) {
        super(ShippingQuery.class, validator);
    }
}

File: core/src/main/kotlin/io/github/oleksivio/tl/kbot/core/model/annotation/validator/impl/ShippingQueryFilterValidator.kt
package io.github.oleksivio.tl.kbot.core.model.annotation.validator.impl

import io.github.oleksivio.tl.kbot.core.controller.handler.check.Validator
import io.github.oleksivio.tl.kbot.core.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.payments.ShippingQuery

class ShippingQueryFilterValidator(validator: Validator<ShippingQuery>) : FilterValidator<ShippingQuery>(
    ShippingQuery::class, validator
)
